cookbook/assetic/asset_management.rst

```diff
@@ -204,7 +204,7 @@ Combining Assets
 ~~~~~~~~~~~~~~~~
 
 One feature of Assetic is that it will combine many files into one. This helps
-to reduce the number of HTTP requests, which is great for front end performance.
+to reduce the number of HTTP requests, which is great for frontend performance.
```

Comment:
Not sure about this change. While I agree that "frontend" looks most beautiful, it seems that "front-end" actually would be more correct (see also https://github.com/symfony/symfony-docs/pull/4671#discussion_r26449893).